packages/cli/src/commands/mock.ts

```diff
@@ -30,13 +43,30 @@ const mockCommand: CommandModule = {
       cors,
       document,
       errors,
+      callbackDelay,
+      callbackCount,
     } = (parsedArgs as unknown) as CreateMockServerOptions;
 
     const createPrism = multiprocess ? createMultiProcessPrism : createSingleProcessPrism;
-    const options = { cors, dynamic, port, host, document, multiprocess, errors };
+    const options = { cors, dynamic, port, host, document, multiprocess, errors, callbackDelay, callbackCount };
 
     return runPrismAndSetupWatcher(createPrism, options);
   },
 };
 
+function coerceRange(input: string): RangeOrNumber {
+  const matches = /^([0-9]+)(-([0-9]+))?$/.exec(input);
```

Comment:
Ok I've dug into and there's a way to specify a *NON* capturing group:
```suggestion
  const matches = /^([0-9]+)(?:-([0-9]+))?$/.exec(input);  
```